minicluster/src/main/java/org/apache/accumulo/cluster/ClusterControl.java

```diff
@@ -49,12 +50,13 @@ public interface ClusterControl {
   /**
    * Starts all occurrences of the given server
    */
-  void startAllServers(ServerType server) throws IOException;
+  void startAllServers(ServerType server) throws InterruptedException, IOException, KeeperException;
 
   /**
    * Start the given process on the host
    */
-  void start(ServerType server, String hostname) throws IOException;
+  void start(ServerType server, String hostname)
+      throws InterruptedException, IOException, KeeperException;
```

Comment:
Do we need to propagate all the ZK exceptions? They seem to be leaking everywhere throughout our code, and I think we will probably need to do a better job of reining these in.

Reply:
Removed the zk exceptions in 258a2e68165a8dc78eb5d47da2374f2ccfec69f9